### ISSUE TITLE
eml dataTable

### ISSUE THREAD

**Issue by mbjones** [NCEAS/eml#65]
---

Author Name: **Owen Eddins** (Owen Eddins)
Original Redmine Issue: 539, https://projects.ecoinformatics.org/ecoinfo/issues/539
Original Date: 2002-06-27
Original Assignee: Matt Jones

---

I'm passing the following comments from Tim Bergsma the data manager at Kellog 
Biological Station in Michagen.  He made them in a eml-dev email.  I posting to 
bugzilla just to make sure they don't fall through the cracks.

12.  An <alternateIdentifier> for <dataTable> would be useful.
13.  An <additionalMetadata> for <dataTable> would be useful.  Several
of us have a 'Comments' field associated with our dataTables, but no
natural place to put them.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Matt Jones** (Matt Jones)
Original Date: 2002-08-20T22:44:12Z

---

I have no problem with either of these suggestions, and they could be useful. 
If nobody objects, I'll go ahead and add the suggested fields as
optional/repeatable.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Matt Jones** (Matt Jones)
Original Date: 2002-09-02T16:16:45Z

---

Added two new optional fields, <alternateIdentifier> and <additionalInfo>.  I
chose the name for the latter based on the name used for the comments field in
the resource module.  I added the fields to entity rather than data table
because it seemed to me they would be useful in all entities, not just
dataTables as originally requested by Tim. This should take care of Tim's two
requests for dataTable.
FIXED.
